_data/language.yaml

```diff
@@ -23,3 +23,7 @@ array:
   - name: C++
     url: //github.com/opentracing/opentracing-cpp
     icon: ''
+
+  - name: Contributions
+    url: //github.com/opentracing-contrib
```

Comment:
But the title is `Libraries available in 6 languages`, and `Contributions` under this, seems strange? @pavolloffay .
`Contributions` contains something about using OT-x-languages, not add a new supported languages.

I prefer to add `Contributions` at the homepage top menu, maybe after `BLOG`.